test/integration/src/test/java/io/pravega/test/integration/IdleSegmentTest.java

```diff
@@ -56,7 +56,7 @@ public void teardown() {
         this.serviceBuilder.close();
     }
 
-    @Test(timeout = 5000)
+    @Test(timeout = 20000)
```

Comment:
This seems to be an unrelated change

Reply:
This single test times out systematically on my laptop when running the suite on jdk10.
I did not find the root cause. 
Maybe it will be better to dig into the problem.
I am still new to Pravega internals.
It will take time to debug the real cause of the slowdown of the test, as it is all async and CompletableFuture based (great work!)
Some help from an expert will be really appreciated